fecgov/openFEC
# switch to use fecp driven sched_c, sched_d, sched_e, sched_f tables

## Summary (required)

#3089
Switching API to the fecp-driven sched tables: schedule c, sched_d, schedule e, schedule f tables and related objects

## How to test the changes locally

After download the branch to local machine, run migration file on local database.  The following columns should be added:
disclosure.fec_fitem_sched_f:
payee_name_text

disclosure.fec_fitem_sched_c:
candidate_name_text
loan_source_name_text

disclosure.fec_fitem_sched_d:
creditor_debtor_name_text

The source of public.ofec_sched_e_aggregate_candidate_mv should be on FECP driven tables instead of views:
fec_fitem_sched_e_vw => disclosure.fec_fitem_sched_e 
fec_fitem_f57_vw => disclosure.fec_fitem_f57
 
Viewing on API, the source for 
http://localhost:5000/developers/#/party-coordinated%20expenditures
should be disclosure.fec_fitem_sched_f instead of public.ofec_sched_f_mv

http://localhost:5000/developers/#/loans
should be disclosure.fec_fitem_sched_c instead of public.ofec_sched_c_mv

http://localhost:5000/developers/#/debts
should be disclousre.fec.fitem_sched_d instead of public.fec_fitem_sched_d_vw

## Impacted areas of the application
List general components of the application that this PR will affect:
The change involved the following three areas.  However, the change should be transparent to the users.

https://api.open.fec.gov/developers/#/operations/independent%20expenditures/get_committee__committee_id__schedules_schedule_e_by_candidate_
https://api.open.fec.gov/developers/#/party-coordinated%20expenditures
https://api.open.fec.gov/developers/#/loans
https://api.open.fec.gov/developers/#/debts

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
